src/js/index.js

```diff
@@ -1,3 +1,7 @@
 import videojs from './video';
 import '@videojs/http-streaming';
+
+// Prefer to use MSE for playback over native
+videojs.options.hls.overrideNative = !videojs.browser.IS_ANY_SAFARI;
```

Comment:
The default options should probably go here: https://github.com/videojs/video.js/blob/master/src/js/player.js#L3752